elytron/WFCORE-4496-principal-transformer-in-aggregate-realm.adoc

```diff
@@ -0,0 +1,71 @@
+= [WFCORE-4496] Adding principal-transformer in aggregate-realm
+:author:            Ashley Abdel-Sayed
+:email:             aabdelsa@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+Elytron currently has an aggregate security realm which is a combination of two or more realms: an authentication realm
+and one or more authorization realms. There is currently no way to transform the principal in between loading the authentication
+identity and loading the authorization identity.
+
```

Comment:
Can you please write short sentence/s why we want between authentication and authorization realms the principal-transformer?
(high-level view, Customer user case)

Reply:
Added

Reply:
+1